frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -193,6 +206,30 @@ export const PaymentInput = (): JSX.Element => {
 
         {watchedEnabled && (
           <>
+            <FormControl
+              isReadOnly={paymentsMutation.isLoading}
+              isInvalid={!!errors.description}
+              isRequired
+            >
+              <Flex>
+                <FormLabel>Description</FormLabel>
+                <Spacer />
+                <Tooltip
+                  label={DESCRIPTION_TOOLTIP}
+                  placement="top"
+                  textAlign="center"
+                >
+                  <Icon as={BxsHelpCircle} aria-hidden ml="0.5rem" />
+                </Tooltip>
+              </Flex>
+              <Textarea
```

Comment:
Change `Textarea` to `Input` as shown in @staceytan1998's design to convey the rough text length expected.

Reply:
Thanks! [63d3568](https://github.com/opengovsg/FormSG/pull/5998/commits/63d3568194fa66f0c71f18a6bf55cd20f0e19515)